### ISSUE TITLE
minimum layer thickness

### ISSUE THREAD

**Issue by tiralonghipol** [vapetrov/PCB_strain_gauge#1]
amazing project! thanks for open sourcing this

what would be the minimum thickness of the plate? I am looking for something like 45 / 60 degree bending, more like a flex sensor
do you think it is possible? sorry if the question might sounds dumb, my knowledge in pcb design is very minor

**Comment by vapetrov** [comment]
Signal strength is proportional to thickness. In other words, if you make it half as thick you will get half the output signal (at least theoretically, it might be less than half in practice). If you do make it very thin, make sure to reinforce the area underneath the solder joins to prevent cracking.

**Comment by tiralonghipol** [comment]
@vapetrov  thank you for the fast reply!
as you wrote in the readme, but just to confirm: to minimize weight I can leave out U6, U4 and U8 from the assembly and read with my own external adc on the output pins, is that correct?

**Comment by vapetrov** [comment]
Yes, with some caveats. If you use your own external ADC, the zero point will be at half the supply voltage (1.65v) so your readings might shift around if the supply voltage changes. If you omit the biasing circuitry, you might end up with a large bias voltage when the sensor is at rest. It really depends on how lucky you get with the tolerances of the layer thickness from the manufacturer. They don't guarantee you will get *exactly* 1oz/35µm, and 1µ in either direction will throw off the readings entirely.

I suggest building it as-is and removing stuff one by one to see how it would affect the quality of your readings.

**Comment by tiralonghipol** [comment]
thank you again, I will give it a try!
